rosidl_typesupport_c/README.md

```diff
@@ -0,0 +1,6 @@
+# rosidl_typesupport_c
+
+`rosidl_typesupport_c` provides functionality for getting the associated message or service c typesupport handler functions.
+
+* `message_type_support_dispatch.h`: Type support handling for ROSIDL messages
+* `service_type_support_dispatch.h`: Type support handling for ROSIDL services
```

Comment:
"Type support handling" doesn't say much what this function is doing / used for.

Also it would be god to reference the header with its full include directory.

Reply:
Updated sentence to reflect actual functionality.